### PR TITLE
#42 review components

### PR DESCRIPTION
**Pull request by roboMoose**
The Reviews back-end is complete. 

Fixed the JEST test problem finally (THANKS ELIOT!). 

Please remember to git pull and update with your current branch. 

The index file in the server folder is important; it looks silly but please dont remove.

### CODE REVIEW

File: server/db/models/Review.js [hunk 1]
```diff
@@ -1,12 +1,24 @@
...
+  },
+  author: {
+    type: Sequelize.STRING
+  },
   body: {
     type: Sequelize.TEXT,
     allowNull: false,
     validate: {
-      len: [10,250]
+      len: [10, 500]
```
Comment (nicknapoli82) [inline server/db/models/Review.js]: How about we don't impose a limit on this? Reviews should be allows to be expressive I think. This isn't Twitter...

Comment (roboMoose) [reply server/db/models/Review.js]: You want them longer? Thanks cool - I'll take the limit off and have it set to type.text on the next merge

File: server/routes/reviews.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
+const router = require("express").Router();
+const { User, Product, Review } = require("../db/index.js");
+
+//All reviews
```
Comment (nicknapoli82) [inline server/routes/reviews.js]: Why would we ever send all reviews ever made to the client? This route feels confusing to me...

File: server/routes/reviews.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+router.get("/", async (req, res, next) => {
+  try {
+    res.send(await Review.findAll());
+  } catch (err) {
+    next(err);
+  }
+});
+
+//Single review
+router.get("/:id", async (req, res, next) => {
```
Comment (nicknapoli82) [inline server/routes/reviews.js]: can we qualify what id means?
I feel that I can guess that this req.params.id is the productId. I'm just not sure.
So maybe "/:productId"

Comment (nicknapoli82) [reply server/routes/reviews.js]: Oh... I read your comment. So a single review. Can we make that explicit

**Pull request merged**